conf/messages

```diff
@@ -232,15 +232,16 @@ select-reason-for-claim.reason.d10=Proof of returns/refund given
 
 
 #===================================================
-#  BANK DETAILS PAGE - START
+#  CHECK BANK DETAILS PAGE - START
 #===================================================
 
 bank-details.title=Check these bank details are correct
 bank-details.caption=Bank Details
 bank-details.help-text=These are the details of where your reimbursements will be made in the UK based on our records.
+bank-details.different-details-link=Enter different bank details
 
 #===================================================
-#  BANK DETAILS PAGE - END
+#  CHECK BANK DETAILS PAGE - END
 #===================================================
```

Comment:
if you look at the latest version of the messages file, I have removed all the "END" blocks - it was visually jarring. There's only a main comment block for the start of the messages for a page. So this END block can be removed now